vyper/venom/basicblock.py

```diff
@@ -126,6 +126,11 @@ def __eq__(self, other) -> bool:
             return False
         return self.value == other.value
 
+    def same(self, other) -> bool:
+        if not isinstance(other, type(self)):
+            return False
+        return self.value == other.value
```

Comment:
identical to `__eq__()`. we should either remove it, or at least reduce the implementation of either `__eq__()` or `same()` to call the other